test/pti.jl

```diff
@@ -237,3 +237,238 @@ TESTLOG = Memento.getlogger(PowerModels)
         # @test parsed_pti == pti
     end
 end
+
+
+@testset "test idempotent pti export" begin
+
+    function test_pti_idempotent(filename::AbstractString, parse_file::Function)
+        source_data = parse_file(filename)
+
+        file_tmp = "../test/data/tmp.raw"
+        PowerModels.export_pti(file_tmp, source_data)
+
+        destination_data = PowerModels.parse_file(file_tmp)
+        rm(file_tmp)
+
+        # Delete "name"  key
+        delete!(source_data, "name")
+        delete!(destination_data, "name")
```

Comment:
Curious, what causes the name to change?

Reply:
Because it makes a temporary file `tmp.raw` so the name of the destination data is `tmp`. For the source data is `filename`. That is the difference.

Reply:
thanks, makes sence.